lib/generators/geoblacklight/templates/settings.yml

```diff
@@ -1,8 +1,8 @@
 # Configurable Logo Used for CartoDB export
 APPLICATION_LOGO_URL: 'http://geoblacklight.org/images/geoblacklight-logo.png'
 
-# CartoDB OneClick Service https://cartodb.com/open-in-cartodb/
-CARTODB_ONECLICK_LINK: 'http://oneclick.cartodb.com/'
+# Carto OneClick Service https://carto.com/engine/open-in-carto/
+CARTO_ONECLICK_LINK: 'http://oneclick.cartodb.com/'
```

Comment:
Hmm... maybe? I was going off of the documentation at https://carto.com/engine/open-in-carto/

![screen shot 2016-08-05 at 11 34 05 am](https://cloud.githubusercontent.com/assets/1656824/17441752/8e1e86b6-5b00-11e6-93ef-90d4252a2165.png)